planner/core/plan.go

```diff
@@ -310,13 +321,59 @@ func (p *basePhysicalPlan) ExtractCorrelatedCols() []*expression.CorrelatedColum
 	return nil
 }
 
+func (p *baseLogicalPlan) GetBakTimeStamp() uint64 {
```

Comment:
Better to rename this function, since timestamp is pretty confusing now.

Reply:
Do U have any suggestion? @eurekaka

Reply:
`logicalTS4TaskMapReset()`? also please add comment to explain the usage of this value.